test/system/moderation_tests.rb

```diff
@@ -27,13 +27,13 @@ def setup
   test "moderating and unmoderating a user" do
     visit "/profile/#{users(:bob).username}"
 
-    find("a#info-ellipsis").click
+    find("a#info-ellipsis").click()
```

Comment:
Are `()` required in ruby? just curious!

Reply:
No they aren't but @VladimirMikulic thinks it's a bit more readable.

Reply:
ok!!